templates/vue/src/store/mutations.js

```diff
@@ -111,19 +111,31 @@ export function deleteLink(state, { source, target }) {
   )
 }
 
-// quizzes
-export function completeQuestion(state, { nodeId, questionId }) {
+// Activities
```

Comment:
Please keep as lower case since other parts of this file's comments are using lowercase.